ts/components/PreferenceItem.tsx

```diff
@@ -29,11 +28,11 @@ type Props = ValueProps | ActionProps;
  */
 class PreferenceItem extends React.Component<Props> {
   public render() {
+    // FIXME: this brokes the TS type matching and types refinement.
```

Comment:
what does it mean?

Reply:
My fault, I did a wrong minor refactoring.